ecs-cli/modules/config/params.go

```diff
@@ -52,23 +52,14 @@ func recursiveFlagSearch(context *cli.Context, flag string) string {
 
 // NewCLIParams creates a new ECSParams object from the config file.
 func NewCLIParams(context *cli.Context, rdwr ReadWriter) (*CLIParams, error) {
-	ecsConfig, configMap, err := rdwr.GetConfig()
+	clusterConfigFlag := recursiveFlagSearch(context, ecscli.ClusterConfigFlag)
```

Comment:
nit:
What do you think of renaming this to just `clusterConfig` and `profileConfig`